minify_web_js.py

```diff
@@ -57,7 +57,8 @@ def minify_closure(file_in, file_out):
 elif module_exists('slimit'):
     from slimit import minify
 else:
-    raise ImportError('Requires "slimit" package for minifying WebUI files.')
```

Comment:
Could a warning be printed that the files are not being minified

Reply:
yeah, sure. I'll add it.